src/AmplifyGeofenceControl/index.ts

```diff
@@ -395,13 +336,7 @@ export class AmplifyGeofenceControl {
     }
     this._editingGeofenceId = id;
 
-    const mapBounds = this._map.getBounds();
-    const polygon = getPolygonFromBounds(mapBounds);
-    const data: Feature = {
-      id: this._editingGeofenceId,
-      ...getGeofenceFeatureFromPolygon(polygon),
-    };
-    this._addToMapboxDraw(data);
+    this._amplifyDraw.drawCircularGeofence(id);
```

Comment:
Is this because the default mode is circular geofence?

Reply:
Yep based on the mocks we should default to circular